packages/expo-sqlite/android/build.gradle

```diff
@@ -82,5 +82,7 @@ android {
 dependencies {
   implementation project(':expo-modules-core')
 
+  implementation 'com.github.requery:sqlite-android:3.39.2'
```

Comment:
Probably we want to play safe with version upgrades, but I'm wondering if it would be beneficial to use the most recent SQLite version available, as it brings [several performance improvements and features](https://www.sqlite.org/changes.html) (like JSON accessors with `->`).